SDK/MobileCenter/Microsoft.Azure.Mobile.Android/MobileCenter.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.Mobile
     /// <summary>
     /// SDK core used to initialize, start and control specific service.
     /// </summary>
-    public static partial class MobileCenter
+    public partial class MobileCenter
```

Comment:
Done. And it is no longer static because it is implemented as a singleton for Windows